nilearn/datasets.py

```diff
@@ -26,6 +25,15 @@
 
 import nibabel
 
+# Define _get_time, which is used module-wide to get the current time.
+try:
+    # Use uptime (when available) to avoid reporting time while computer slept.
+    import uptime
+    _get_time = uptime.uptime
```

Comment:
This is a tricky question... If uptime calculation is not supported on a system, will it fail at import or at function call? Apart from that, this PR is ready to go.